Makefile.pre.in

```diff
@@ -290,6 +290,9 @@ HOSTRUNNER= @HOSTRUNNER@
 PYTHON_FOR_REGEN?=@PYTHON_FOR_REGEN@
 UPDATE_FILE=$(PYTHON_FOR_REGEN) $(srcdir)/Tools/scripts/update_file.py
 PYTHON_FOR_BUILD=@PYTHON_FOR_BUILD@
+# Normal builds depend on $(BUILDPYTHON). Cross builds use an external
```

Comment:
I think "normal" is too abstract. Can it be clarified like this?

```suggestion
# Same-platform builds depend on $(BUILDPYTHON). Cross builds use an external
```

Reply:
How else would you call standard, non-cross-builds?

Reply:
Single-platform seems to be the common term for non-cross-builds.